pkg/kf/internal/testutil/assertions.go

```diff
@@ -125,3 +126,20 @@ func AssertKeyWithValue(t Failable, m map[interface{}]interface{}, key, value in
 		t.Fatalf("expected %v to have key %v with value %v", m, key, value)
 	}
 }
+
+// AssertJSON ensures the expexted JSON matches the actual JSON.
+func AssertJSON(t Failable, expected, actual string) {
```

Comment:
I think this should be `AssertJSONEqual(s)`, also consider taking a type of `json.RawMessage`

Reply:
Agreed with the name.

Can you expand on the benefit of `json.RawMessage`? It seems like that will require tests using it to have to do type casting from a string.